packages/web-runtime/src/index.ts

```diff
@@ -32,20 +32,21 @@ export const bootstrap = async (configurationPath: string): Promise<void> => {
   const runtimeConfiguration = await announceConfiguration(configurationPath)
   startSentry(runtimeConfiguration, Vue)
   await announceStore({ vue: Vue, store, runtimeConfiguration })
-  await initializeApplications({
+  const applications = initializeApplications({
     runtimeConfiguration,
     store,
     supportedLanguages,
     router,
     translations
   })
+  const theme = announceTheme({ store, vue: Vue, designSystem, runtimeConfiguration })
   announceClientService({ vue: Vue, runtimeConfiguration })
+  await announceClient(runtimeConfiguration)
+  const authService = announceAuthService({ vue: Vue, configurationManager, store, router }) // Requires client
+  await Promise.allSettled([applications, authService, theme])
```

Comment:
Please use `Promise.all`, as `allSettled` would swallow any errors. But we want to show the error page If any of the three promises fails, so needs to be `Promise.all`.